libexec/goenv-prefix

```diff
@@ -16,7 +16,8 @@ if [ "$1" = "--complete" ]; then
 fi
 
 versions() {
-  goenv versions --bare | $(type -p ggrep grep | head -1) -F "$query" || true
+  # Sort correctly (1.20.9 comes before 1.20.10)
+  goenv versions --bare | sort -V | $(type -p ggrep grep | head -1) -F "$query" || true
```

Comment:
Thanks for fixing the sort here. I think this was fixed in other places but this was missed

Reply:
Just saw it coincidentally because I had such versions installed 😅
But a real fix inside `versions` would be better of course.